_visualisation/core/vocabulary.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Vocabulary and Icons
+position: 1.02
+---
+
+The BOVS "vocabulary" specifies the types of Parties that can appear at the Nodes in a BOVS diagram.
+{: .lead }
+
+
+## Subjects
+
+The following types of Parties are Subjects, and so must appear at the start of all Chains in a diagram.
+
+{% capture PersonInfo %}
+{{ site.data.definitions.BOVS.Person }} Note that the plural in the context of BOVS is "Persons" not "people".
+
+Persons are indicated with the [Person icon](/visualisation/resources).
+{% endcapture %}
+{% capture PersonInfo %}{{ PersonInfo | markdownify }}{% endcapture %}
+
+{% capture RegimeInfo %}
+{{ site.data.definitions.BOVS.Regime }}
```

Comment:
I think this might also need input from @scatteredink as it relates to work in progress on State Owned Enterprises - but my sense seeing this now is that we should talk here about **State bodies** rather than **Regimes**.

We'll then need to:

* Highlight that "In some cases, an entity may be owned or controlled by a state body in a way that does not allow identification of specific Persons who have ownership or control. This can be shown by (a) giving the name of the state body that has ownership or control; (b) using a flag to indicate the jurisdiction of that state body." 

We might not want to fully specify ISO 3166-1 for jursidiction codes - as I could imagine a diagram context in, say the US, where using State flags is preferred to using the national flag.

Reply:
A second issue here that I think needs discussion with BODS team is on 'Listed companies' - as that is the other kind of 'Interested Party' that can, in practice, terminate an ownership chain.

Reply:
Yes - we need to allow that listed companies can terminate ownership chains.

Reply:
This makes sense @kd-ods @timgdavies. To clarify how this works, if an Entity is owned by Airstrip One, an administrative division within the nation of [Oceania](https://en.wikipedia.org/wiki/Nations_of_Nineteen_Eighty-Four#Oceania), would we draw a Node with:

* The label "Airstrip One" and the flag of Oceania (label more specific than icon)
* The label Oceania and the flag of Oceania (just identify the state, not the more specific part)
* The label "Airstrip One" and a flag containing the abbreviation for Airstrip One, "A1" (be completely specific)

Reply:
A combination of the first two options, and allowing that it would depend on what data we had or what point we were making. Eg 'Oceania', 'Airstrip One, Oceania', or 'Dept of Transport, UK' - all would be fine in different contexts.)

Reply:
Got it. On the terminology, is "Jurisdiction" better than "State"? – or are all jurisdictions States?

Reply:
All states are jurisdictions.

Not all jurisdictions are states.

Reply:
@timgdavies The benefit of specifying ISO 3166 is consistency and comprehensibility across diagrams. I wouldn't like to see different diagrams all using their own abbreviation scheme.

But we should clarify that it isn't always countries – I will update this. ISO 3166 is useful here again because it also specifies standard codes for the subdivisions of countries (including the US state codes for instance).

There is an edge case where a jurisdiction does not appear anywhere in ISO 3166, although in practice I would expect this is extremely rare.

So my preference would be to say something like:

* If the jurisdiction appears in ISO 3166:
   * If identifying a country, use the 2-letter code from [ISO 3166-1](https://en.wikipedia.org/wiki/ISO_3166-1) (e.g. "US" for the USA)
   * If identifying a subdivision, use the second part (after the dash) of the [ISO 3166-2](https://en.wikipedia.org/wiki/ISO_3166-2) code (e.g. "WA" for Washington)
* Otherwise, either:
   * If the jurisdiction only appears as a Node with a label giving its name, omit an abbreviation within the icon
   * If the jurisdiction appears as Metadata, invent an abbreviation and include a Key

Reply:
We probably ought also to specify that you shouldn't mix and match your specificity level/scope. So, either you identify jurisdictions at the country level, or you do it at the subdivision level and _only if_ all jurisdictions on the diagram are within the same country. Or is that too limiting @kd_ods @timgdavies ?

Reply:
Realised I missed the "Dept of Transport" example. Gah, this is all quite complex isn't it?

A ministry/department is sort of an Entity, sort of a jurisdiction, in that it's an entirely state-operated/linked body. How should this be shown? —

* United Kingdom (jurisdiction icon) →owns/controls→ Dept of Transport (Entity icon) →owns→ HS2 Ltd. (Entity icon)
* Dept of Transport (jurisdiction icon) →owns→ HS2 Ltd. (Entity icon)

I'd prefer the former just because it's more consistent with our approach to the icons/vocabulary – but of course this must be in service of what you guys want to do and what actually makes sense. What do you think?

Reply:
@Convincible If possible, I'd leave this level of specificity out at the moment, re ISO 3166-2 requirement and re how to model internal state dynamics. Only because these are issues that are being worked out in practice as BODS gets developed & used.

Reply:
Thanks @kd-ods. Are we settled, given @timgdavies's comment above, on "Jurisdiction" for this type of Party?

Reply:
@Convincible - I don't think 'jurisdiction' or 'state' is is perfect, but - semantically - 'state' is preferable. (Since 'jurisdiction' really means the scope or reach of the law, territorial or otherwise, or it refers to the power to enforce the law. 'State' can happily encompass state bodies and authorities which are relatively tangible. And ultimately, in this domain, it's the state which we can point to as - e.g. - the ultimate owner of National Rail. Saying that a jurisdiction owns or controls a company doesn't really make sense.)